docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -136,14 +136,16 @@ Use the Grafana Alerting - Grafana OnCall integration to effortlessly connect al
 
 To learn more, refer to the [Grafana OnCall integration for Alerting documentation](/docs/grafana/next/alerting/alerting-rules/manage-contact-points/configure-oncall/).
 
-## New browse dashboards view
+## New Browse Dashboards
 
 <!-- Yaelle Chaudy for Frontend Platform -->
 <!-- OSS, Enterprise -->
 
-_Available in public preview in Grafana Cloud_
+_Generally available in Grafana Cloud_
+
+The new Browse Dashboards interface features a more compact design, making it easier to navigate, search for, and manage for your folders and dashboards. The new interface also has many performance improvements, especially for instances with a large number of folders and dashboards.
```

Comment:
I don't see anything in the UI with this name and this isn't a product name, so it should be lower case
```suggestion
The new browse dashboards interface features a more compact design, making it easier to navigate, search for, and manage for your folders and dashboards. The new interface also has many performance improvements, especially for instances with a large number of folders and dashboards.
```